### DISCUSSION TITLE
Realidade Aumentada na web

### DISCUSSION THREAD

**Discussion by lidymonteirowm** [frontendbr/forum#730] [Perguntas e Respostas]
Estive pesquisando e encontrei esses projetos voltados para AR. 
Ar.js: https://github.com/jeromeetienne/AR.js 
https://github.com/artoolkit/jsartoolkit5

Desejo utilizar junto com [A-Frame da Mozilla (VR)](https://aframe.io/) e o Ar.js é um projeto q parece ser mais ativo e com recomendação do proprio a-frame.
Porém, há poucos tutoriais. VR está mais avançado na web que AR. Vocês já tiveram essa demanda de AR com Web?

Poderiam me dar uma luz do que estudar do javascript (um roteiro de estudo) para dominar tree.js,  jsartoolkit e ar.js?

Sei que tem outras soluções para AR mais desenvolvidas como Unity + Vuforia, mas gostaria de ver algo pra web. Tenho mais contato com web e talvez seja mais fácil integrar com o que ja trabalho. 

Abraço

**Comment by afonsopacifer** [comment]
Sobre three.js,
fiz um video  e artigo passando pelos conceitos básicos:

Artigo:
https://codepen.io/afonsopacifer/post/introducao-a-modelagem-3d-com-three-js

Video:
https://www.youtube.com/watch?v=sZypsYCIXIQ

Fora isso.. só sei sobre como utilizar A-frame +  three.js com Web components.. se quiser saber mais, teve uma talk legal sobre no último Polymer Summit:

https://www.youtube.com/watch?v=8GmTu2JF4-0

**Comment by kivervinicius** [comment]
o A-Frame vi na BrazilJS está muitoo show!

**Comment by yanmagale** [comment]
Verdade @kivervinicius , pelo o que eles mostraram parece ser mto mto massa.
